lib/block-supports/block-style-variations.php

```diff
@@ -394,17 +412,17 @@ function gutenberg_enqueue_block_style_variation_styles() {
 	remove_action( 'wp_enqueue_scripts', 'wp_enqueue_block_style_variation_styles' );
 }
 
-if ( function_exists( 'wp_resolve_block_style_variations_from_primary_theme_json' ) ) {
-	remove_filter( 'wp_theme_json_data_theme', 'wp_resolve_block_style_variations_from_primary_theme_json' );
+if ( function_exists( 'wp_resolve_block_style_variations_from_theme_json_default' ) ) {
+	remove_filter( 'wp_theme_json_data_theme', 'wp_resolve_block_style_variations_from_theme_json_default' );
```

Comment:
I think there's a copy paste error here, it should be `wp_theme_json_data_default` right?